client/js/libs/handlebars/tz.js

```diff
@@ -1,8 +1,10 @@
+var timestamp = window.timestamp;
```

Comment:
I'd suggest keeping the variable instead of the shout.js's anon function scope. But if you're keeping it in `window`, then you should give it a better name (ideally with a namespace).